crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -912,18 +914,18 @@ const NU_PLUGIN_DIRS = $NU_PLUGIN_DIRS ++ [($nu.default-config-dir | path join '
 
 # Appending to the OS path is a common configuration task.
 # Because of the previous ENV_CONVERSIONS (performed internally
-# before your config.nu loads), the path variable is a list that can 
+# before your config.nu loads), the path variable is a list that can
 # be appended to using, for example:
-$env.path ++= "~/.local/bin"
+$env.PATH ++ [ "~/.local/bin" ]
 
 # Or prepend using
-$env.path = "~/.local/bin" ++ $env.path
+$env.PATH = [ "~/.local/bin" ] ++ $env.PATH
 
 # The `path add` function from the Standard Library also provides
 # a convenience method for prepending to the path:
 use std/util "path add"
 path add "~/.local/bin"
-path add ($env.CARGO_HOME | path join "bin")
+path add ($env.HOME | path join ".cargo/bin")
```

Comment:
I don't have cargo installed, so I made it to not depend on cargo

Reply:
Well, that would break on my system ;-)

It's okay to have *examples* that don't run on everyone's system.  The point here is a dynamically computed path based on an environment variable.  Otherwise, you could simply write:

```
path add "~/.cargo/bin"
```

That's the exact same example as the one above it ;-)